plugins/base/src/main/kotlin/translators/psi/parsers/JavadocParser.kt

```diff
@@ -334,7 +333,8 @@ class JavadocParser(
         ) =
             when (tag.name) {
                 "link", "linkplain" -> tag.referenceElement()
-                    ?.toDocumentationLinkString(tag.dataElements.filterIsInstance<PsiDocToken>())
+                    ?.toDocumentationLinkString(tag.dataElements.filterIsInstance<PsiDocToken>().joinToString("") { it.stringifyElementAsText(keepFormatting = true)
+                        .toString() })
```

Comment:
use `.orEmpty` or something like this since if you use `toString` on `null` it will be displayed as null